### PR TITLE
Re add Oscars 2024 Knowledge

### PR DESCRIPTION
**Pull request by juliadenham**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file.

### CODE REVIEW

File: knowledge/pop_culture/oscar_results/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+created_by: juliadenham
+domain: pop_culture
+seed_examples:
+  - answer: |
+      The 2024 oscars were held on March 10, 2024.
```
Comment (mingxzhao) [inline knowledge/pop_culture/oscar_results/qna.yaml:5]: Capital O Oscars :)

**Comment by RobotSail** [comment]
To fix this PR run `git rebase -i upstream/main` and set the operations to:

```
drop 1966cef feat(synthetic data): Add script to generate synthetic data readme (#650)
drop cc381ec Updating org name from instruct-lab to instructlab (#711)
pick 2b6c3b1 Create qna.yaml
pick 0566a75 Update knowledge/pop_culture/oscar_results/qna.yaml
```

File: knowledge/pop_culture/oscar_results/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    question: When did the 2024 oscars happen?
+  - answer: |
+      Oppenheimer had 13 oscar nominations.
+    question: What film had the most oscar nominations in 2024?
+  - answer: |
+      Octavia Spencer.
```
Comment (aakankshaduggal) [inline knowledge/pop_culture/oscar_results/qna.yaml:11]:
```suggestion
      Octavia Spencer presented the awards for Best Original Screenplay and Best Adapted Screenplay.
```

File: knowledge/pop_culture/oscar_results/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+seed_examples:
+  - answer: |
+      The 2024 oscars were held on March 10, 2024.
+    question: When did the 2024 oscars happen?
+  - answer: |
+      Oppenheimer had 13 oscar nominations.
+    question: What film had the most oscar nominations in 2024?
+  - answer: |
+      Octavia Spencer presented the awards for Best Original Screenplay and Best Adapted Screenplay.
+    question: Who presented the awards for Best Original Screenplay and Best Adapted Screenplay?
```
Comment (aakankshaduggal) [inline knowledge/pop_culture/oscar_results/qna.yaml:12]:
```suggestion
    question: Who presented the 2024 Oscar awards for Best Original Screenplay and Best Adapted Screenplay?
```

File: knowledge/pop_culture/oscar_results/qna.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      Yorgos Lanthimos for Poor Things, and Jonathan Glazer for The Zone of Interest.
+      Christopher Nolan won best director for Oppenheimer.
+    question: At the 2024 oscars, who were the nominees for best director and who won?
+  - answer: |
+      Yes Billie Eilish performed "What Was I Made For?" from Barbie.
+    question: Did Billie Eilish perform at the 2024 oscars?
+task_description: 'Teach the model the results of the 2024 oscars'
+document:
+  repo: https://github.com/juliadenham/oscars2024_knowledge
+  commit: a22148c
```
Comment (aakankshaduggal) [inline knowledge/pop_culture/oscar_results/qna.yaml:28]:
```suggestion
  commit: e1744af
```

**Comment by jjasghar** [comment]
Can we close this one?

**Comment by alinaryan** [comment]
Thank you for your submission. Here are your results: 
Your PR improved the model performance by 0.013333333333333308 points. 
The model's new score is: 0.26666666666666666 
 The old score was: 0.25333333333333335 
||||||||||||||||||||||||||| 26.666666666666668%

**Pull request closed without merging**